bundles/org.openhab.binding.roku/src/main/resources/OH-INF/thing/roku.xml

```diff
@@ -13,7 +13,9 @@
 
```

Comment:
I did some additional testing... When adding a new thing, a duplicate channel error is seen due to the missing **thingTypeVersion**.

```suggestion
		<properties>
			<property name="thingTypeVersion">1</property>
		</properties>
```
See this for more info: https://www.openhab.org/docs/developer/bindings/thing-xml.html#updating-thing-types